src/main/java/com/github/shiranr/scooters/domain/internal/Scooter.java

```diff
@@ -0,0 +1,66 @@
+package com.github.shiranr.scooters.domain.internal;
+
+import java.sql.Time;
+import java.util.ArrayList;
+import java.util.Date;
+
+/**
+ * This is a scooter entity, it represents scooter both in the DB and FE.
+ * In the future, should be separated into 2 entities so the DB scheme and FE scheme won't be coupled.
+ */
+public class Scooter {
```

Comment:
isn't this a duplication of src/main/java/com/github/shiranr/scooters/domain/Scooter.java ?

Reply:
Yes, I thought about separating the DB entity from the FE. Not sure it's really needed since it's an exercise.